src/containers/shared/components/Transaction/AMMWithdraw/index.ts

```diff
@@ -1,8 +1,14 @@
 import { Simple } from './Simple'
-import { TransactionMapping } from '../types'
+import {
+  TransactionAction,
+  TransactionCategory,
+  TransactionMapping,
+} from '../types'
 import { parser } from './parser'
 
 export const AMMWithdraw: TransactionMapping = {
   Simple,
+  action: TransactionAction.SEND,
   parser,
+  category: TransactionCategory.DEX,
```

Comment:
We should be consistent about the order of these elements (doesn't the linter do that?)

Reply:
I can do that.  I am leery of a linter enforcing element order.  It can be difficult when you want to group for documentation purposes.

Reply:
That's fair.